Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -48,18 +51,39 @@ public extension UIConfig {
         public var channelViewCell: ChatChannelListCollectionViewCell<ExtraData>.Type
         public var avatarView: AvatarView.Type
         
+        public var newChannelButton: CreateNewChannelButton.Type
+
         public init(
             channelCollectionView: ChatChannelListCollectionView.Type = ChatChannelListCollectionView.self,
             channelCollectionLayout: UICollectionViewLayout.Type = ChatChannelListCollectionViewLayout.self,
             channelView: ChatChannelView<ExtraData>.Type = ChatChannelView<ExtraData>.self,
             channelViewCell: ChatChannelListCollectionViewCell<ExtraData>.Type = ChatChannelListCollectionViewCell<ExtraData>.self,
-            avatarView: AvatarView.Type = AvatarView.self
+            avatarView: AvatarView.Type = AvatarView.self,
+            newChannelButton: CreateNewChannelButton.Type = CreateNewChannelButton.self
```

Comment:
Yeah, I think it's time to forget about this idea of having everything in the init tbh